dist/pagination/ds4/pagination.css

```diff
@@ -104,38 +103,44 @@
   height: 17px;
   width: 10px;
   background-size: 10px 17px;
+  background-size: contain;
```

Comment:
Is the double background size intended? I know we already had some issues with this across our existing code anyway, but want to make sure this is intended. I do feel like these background size changes should be confined to their own PR (separate from switch, color and checbox refactor).